tests/test_network_modification.py

```diff
@@ -183,12 +171,22 @@ def test_add_load_bay_from_kwargs():
     assert position.direction == 'BOTTOM'
 
 
+def test_add_load_bay_from_kwargs_bus_breaker():
```

Comment:
should it be in test_network_modification ?

Reply:
Why not?

Reply:
my mistake I thought it was in test_network